hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -881,7 +881,7 @@
       tests: ["chip_sw_pwrmgr_full_aon_reset"]
     }
     {
-      name: chip_sw_pwrmgr_sleep_all_wake_ups
+      name: chip_sw_pwrmgr_normal_sleep_all_wake_ups
```

Comment:
What is normal? Can we call this "shallow" or "deep" instead? I also recently created an ambiguously named test :(